reactioncommerce/reaction
# Select Orders

Resolves #2325 

Users with permissions to view orders are able to select/deselect orders within the order list, whether one by one or all the orders listed in the current page.

### How to test

- Clone this branch
- Have at least 3 orders made in your shop
- Visit the orders page on the dashboard
- Hover over an avatar while in the orders list. Notice on hover that the avatar rolls over to a check box
- Select the check box. Notice that a bar appears over the table that has the number of selected orders
- Select the check box in the bar that appears over the table. Notice that all the orders you have listed are now selected. Clicking on the checkbox again deselects all the orders.

N.B: The two buttons that appear on the bar (the `Capture` and `Bulk Actions` buttons) are not functional as at this point. Functionality for them will be handled in #2341
